src/control/server/storage/provider.go

```diff
@@ -346,8 +346,9 @@ func BdevWriteConfigRequestFromConfig(ctx context.Context, log logging.Logger, c
 		// Applying the range limits hotplug activity of engine to a ssd device set.
 
 		var begin, end uint8
-		if tier.Bdev.BusidRange != nil {
-			log.Debugf("received user-specified hotplug bus-id range %q", tier.Bdev.BusidRange)
+		if tier.Bdev.BusidRange != nil && tier.Bdev.BusidRange.String() != "0x00-0x00" {
```

Comment:
When would this `0x00-0x00` range be set? I'm not crazy about this string comparison. I think either we should ferret out where the range is set to this nonsensical (IMO) value and fix that, or at the very least add a `IsZero()` or similar method on `BusidRange` so that we're not relying on fragile string comparisons in production code.

Reply:
I am pretty sure this is a regression that was introduced by this refactor: a4b34159fdaf482a03ae7c248fb2fda628850ae9 . I agree that the string comparison is not great and I'm trying to work out the exact cause.

Reply:
In the commit mentioned above, when config is unmarshaled , by default a bus range  that stringifies into "0x00-0x00" is set, that is the cause of the behavioural regression. I agree that the most straightforward and logical solution is to add IsZero() method to the type, I will make the change.

Reply:
done